README.md

```diff
@@ -24,14 +24,23 @@ The versions required are:
 
 ## Usage
 
-For now, we do not host a chart repository. To use the charts, you must
-download this repository and unpack it into a directory. Either
-[download a tagged release](https://github.com/hashicorp/consul-helm/releases) or
-use `git checkout` to a tagged release.
-Assuming this repository was unpacked into the directory `consul-helm`, the chart can
-then be installed directly:
-
-    helm install ./consul-helm
+Detailed installatiion instructions for Consul on Kubernetes are found [here](https://www.consul.io/docs/k8s/installation/overview#installing-consul). 
```

Comment:
```suggestion
Detailed installatiion instructions for Consul on Kubernetes are found [here](https://www.consul.io/docs/k8s/installation/overview). 
```

Let's link them to the top of that page since it has some stuff worth reading.